content/en/docs/ops/common-problems/security-issues/index.md

```diff
@@ -93,9 +93,13 @@ matching requests should flow through. If all requests continue to be denied, yo
 1. Make sure that your authorization policies with ALLOW action don't use any HTTP only fields for TCP traffic.
 Otherwise, Istio ignores the ALLOW policies as if they don't exist.
 
+1. A HTTP Response with the value `RBAC: Access Denied` indicates an authorization policy is in effect. You can determine the authorization policy in effect by running `istioctl x authz check POD-NAME.POD-NAMESPACE`. This command is available in istioctl v1.7.1+
```

Comment:
Why say it's only available after 1.7.1? The command is also in the 1.6 docs: https://istio.io/v1.6/docs/reference/commands/istioctl/#istioctl-experimental-authz-check. If it wasn't working properly prior to 1.7.1, I don't think we need to highlight that here, since these are the current docs.

```suggestion
1. An HTTP response with the value `RBAC: Access Denied` indicates an authorization policy is in effect. You can determine the authorization policy in effect by running `istioctl x authz check POD-NAME.POD-NAMESPACE`.
```

Reply:
OK, I'll amend it